crates/ln-dlc-node/src/node/storage.rs

```diff
@@ -78,12 +78,10 @@ pub trait Storage {
     fn upsert_channel(&self, channel: Channel) -> Result<()>;
     /// Get channel by `user_channel_id`
     fn get_channel(&self, user_channel_id: &str) -> Result<Option<Channel>>;
-
-    /// Get channel by `user_channel_id`
-    fn get_channel_by_fake_scid(&self, fake_scid: FakeScid) -> Result<Option<Channel>>;
-
     /// Get all non pending channels.
     fn all_non_pending_channels(&self) -> Result<Vec<Channel>>;
+    /// Get announce channel with counterparty
```

Comment:
```suggestion
    /// Get announced channel with counterparty.
```

Also, this is weird because it only applies to the coordinator (I hate the `Storage` trait).